app/src/main/java/com/devindi/wallpaper/settings/size/SizeViewHolder.kt

```diff
@@ -0,0 +1,25 @@
+package com.devindi.wallpaper.settings.size
+
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.widget.TextView
+import com.devindi.wallpaper.settings.model.IntField
+import com.devindi.wallpaper.source.OnItemClickListener
+import kotlinx.android.synthetic.main.settings_simple_item.view.*
+
+class SizeViewHolder(view: View, clickListener: OnItemClickListener) : RecyclerView.ViewHolder(view) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/f22b9ee68b14344acb1bd6cd546c2767866b4eef/app/src/main/java/com/devindi/wallpaper/settings/size/SizeViewHolder.kt#L10"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>